### PR TITLE
Run Tests on CI

### PR DESCRIPTION
**Pull request by zabulus**

### CODE REVIEW

**Comment by zabulus** [comment]
Tests are failing, because refactoring is not added to this commit. It is in PR https://github.com/cincuranet/FirebirdSql.Data.FirebirdClient/pull/29. 
Do I need to fix this PR build?

**Comment by cincuranet** [comment]
No. I'll merge everything anyway.

File: appveyor.yml [hunk 1]
```diff
@@ -1,15 +1,27 @@
...
 os: Default Azure
 configuration:
-- Debug
 - Release_40
 - Release_45
 shallow_clone: true
+matrix: 
+    fast_finish: true
+before_test:
+  - ps: Start-FileDownload 'http://garr.dl.sourceforge.net/project/firebird/firebird-win32/2.5.4-Release/Firebird-2.5.4.26856_0_Win32.exe' | Out-Null
```
Comment (cincuranet) [inline appveyor.yml]: Maybe download the zip and just unpack and run? It might be faster than installation.

Comment (zabulus) [inline appveyor.yml]: Indeed facepalm.jpg.
I'll rework this to unzipping.

Comment (cincuranet) [inline appveyor.yml]: No problem. Thanks.

**Comment by cincuranet** [comment]
OK, I manually merged it [here](https://github.com/cincuranet/FirebirdSql.Data.FirebirdClient/commit/c8ec3f4475f2aa70660dda341f72db93399e4eee) and [here](https://github.com/cincuranet/FirebirdSql.Data.FirebirdClient/commit/3bb1df7796f8328085b532bc18c36681f31a0ce9).

**Pull request closed without merging**